front-end/src/screens/CreatePost/index.tsx

```diff
@@ -55,12 +74,21 @@ const CreatePost = ({ className }:Props): JSX.Element => {
 			.catch((e) => console.error('Error subscribing to post',e));
 	};
 
+	const getEndBlockNumber = (currentBlock: number | undefined): number => {
+		const endBlock = Math.floor(TWO_WEEKS/blocktime);
+
+		return (currentBlock || 0) + endBlock;
```

Comment:
Same here for `|| 0`. If we execute this code without knowing the current block it means there's something wrong before.

Reply:
Added a check for blockNumber